ubcpi/persistence.py

```diff
@@ -0,0 +1,74 @@
+from submissions import api as sub_api
+
+"""
+answer_item has to be in the format:
+{
+    'answers': [
+        {
+            'vote': 'some option',
+            'rationale': 'some rationale'
+        },
+        {
+            'vote': 'some option',
+            'rationale': 'some rationale'
+        }
+    ]
+}
+"""
+
+ANSWER_LIST_KEY = 'answers'
+
+VOTE_KEY = 'vote'
+RATIONALE_KEY = 'rationale'
+
+
+def get_answers_for_student(student_item):
+    submissions = sub_api.get_submissions(student_item)
+    if not submissions:
+        return Answers()
+
+    latest_submission = submissions[0]
+    latest_answer_item = latest_submission.get('answer', {})
+    return Answers(latest_answer_item.get(ANSWER_LIST_KEY, []))
```

Comment:
I'm adding comments to the code as per your previous comments. I think we are using submission API the same as ORA2, by calling get_submissions with student_item dict.